### PR TITLE
Update the README with new information.

### PR DESCRIPTION
**Pull request by Codertocat**
This is a pretty simple change that we need to pull into master.

### CODE REVIEW

**Comment by nmeum** [comment]
Some Comment

**Comment by nmeum** [comment]
Some Comment

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# Hello-World
```
[the remark this answers is not in the document]
Comment (CarlosCondor) [reply README.md:1]: asd

**Comment by johnpapa** [comment]
From all of us on the Cloud Advocacy team at Microsoft, thank you @Codertocat for creating this pull request!

Have a Happy Holiday season!

**Comment by johnpapa** [comment]
From all of us on the Cloud Advocacy team at Microsoft, thank you @Codertocat for creating this pull request!

Have a Happy Holiday season!

**Review by carusogabriel — APPROVED** [review]

**Review by carusogabriel — APPROVED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jalallinux) [reply README.md:1]: asd

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:relevant

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by acccrop** [comment]
```
Error: This repo is not allowlisted for Atlantis.
```

**Comment by acccrop** [comment]
```
Error: This repo is not allowlisted for Atlantis.
```

**Comment by svcosmghtest** [comment]
@Codertocat, you must sign our contributor license agreement before your changes are merged. Click [here](https://d7d132ad.ngrok.io/cla/f594bed8ecfb482d96ae2cde3fa898d8) to sign the agreement. If you are a VMware employee, read [this](https://d7d132ad.ngrok.io/employee) for further instruction.

**Comment by allanwsilva** [comment]
Hello, World!

**Comment by allanwsilva** [comment]
Hello, World!

**Comment by allanwsilva** [comment]
Hello, World!

**Comment by allanwsilva** [comment]
Hello, World!

**Comment by allanwsilva** [comment]
Hello, World!

**Comment by allanwsilva** [comment]
Hello, World!

**Comment by svcosmghtest** [comment]
@BennyThink, VMware has rejected your signed contributor license agreement. The merge can not proceed until the agreement has been resigned. Click [here](http://127.0.0.1:9000/cla/5fbc0d08d0a811ea87d00242ac131111) to resign the agreement.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (iAmNikola) [reply README.md:1]: asd

**Comment by yemonerail** [comment]
Yem was here!

**Review by bbuxton93 — CHANGES_REQUESTED** [review]
Load script test failed.
Log:
ReloadID: 63ed4b2583a3f98255ff416d
Started loading data
(A detailed script progress log can be downloaded when the reload is finished)
Data has not been loaded. Please correct the error and try loading again.
The following error occurred:
Unknown statement: Nobody reads
The error occurred here:
Nobody reads

**Review by bbuxton93 — CHANGES_REQUESTED** [review]
Load script test failed.
Log:
ReloadID: 63ed50ca83a3f98255ff44c3
Started loading data
(A detailed script progress log can be downloaded when the reload is finished)
Data has not been loaded. Please correct the error and try loading again.
The following error occurred:
Unknown statement: Nobody reads
The error occurred here:
Nobody reads

**Comment by jamesrenaud** [comment]
### :robot: *Terraform Change Report*
---

### Change Summary

| Layer Name | Region | Creates | Updates | Deletes |
| :--- |  :--- | :---: | :---: | :---: |
| 20-bootstrap | | 0 | 14 | 0 |
| | uswest | 0 | 7 | 0 |
| | useast | 0 | 7 | 0 |


---


### Terraform Changes

#### **20-bootstrap** ==> uswest
##### Plan: `0` to add, `7` to change, `0` to destroy

<details><summary>Show Plan</summary>

```terraform

Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # module.monitoring.azurerm_resource_group.main_resource_group will be updated in-place
  ~ resource "azurerm_resource_group" "main_resource_group" {
        id       = "/subscriptions/0bf7eeee-63f3-4129-a0bc-b0424afc2633/resourceGroups/rg-kxs-sec-01"
        name     = "rg-kxs-sec-01"
      ~ tags     = {
          + "cat_business_unit" = "cloud_engineering"
          + "cat_environment"   = "shared"
          + "cat_owner"         = "ncuccia"
          + "created_by"        = "terraform"
        }
        # (1 unchanged attribute hidden)

        # (1 unchanged block hidden)
    }

  # module.backend.module.storage_account["stcsp2appeastustfstate"].azurerm_storage_account_network_rules.bootstrap will be updated in-place
  ~ resource "azurerm_storage_account_network_rules" "bootstrap" {
        id                         = "/subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2appeastustfstate"
      ~ ip_rules                   = [
          - "20.121.20.179",
            # (5 unchanged elements hidden)
        ]
        # (4 unchanged attributes hidden)
    }

  # module.backend.module.storage_account["stcsp2cldeastustfstate"].azurerm_storage_account_network_rules.bootstrap will be updated in-place
  ~ resource "azurerm_storage_account_network_rules" "bootstrap" {
        id                         = "/subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2cldeastustfstate"
      ~ ip_rules                   = [
          - "20.121.20.179",
            # (5 unchanged elements hidden)
        ]
        # (4 unchanged attributes hidden)
    }

  # module.backend.module.storage_account["stcsp2fweastustfstate"].azurerm_storage_account_network_rules.bootstrap will be updated in-place
  ~ resource "azurerm_storage_account_network_rules" "bootstrap" {
        id                         = "/subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2fweastustfstate"
      ~ ip_rules                   = [
          - "20.121.20.179",
            # (5 unchanged elements hidden)
        ]
        # (4 unchanged attributes hidden)
    }

  # module.backend.module.storage_account["stcsp2infraeastustfstate"].azurerm_storage_account_network_rules.bootstrap will be updated in-place
  ~ resource "azurerm_storage_account_network_rules" "bootstrap" {
        id                         = "/subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2infraeastustfstate"
      ~ ip_rules                   = [
          - "20.121.20.179",
            # (5 unchanged elements hidden)
        ]
        # (4 unchanged attributes hidden)
    }

  # module.backend.module.storage_account["stcsp2neteastustfstate"].azurerm_storage_account_network_rules.bootstrap will be updated in-place
  ~ resource "azurerm_storage_account_network_rules" "bootstrap" {
        id                         = "/subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2neteastustfstate"
      ~ ip_rules                   = [
          - "20.121.20.179",
            # (5 unchanged elements hidden)
        ]
        # (4 unchanged attributes hidden)
    }

  # module.monitoring.module.loganalytics_workspace.azurerm_log_analytics_workspace.log_analytics_workspace will be updated in-place
  ~ resource "azurerm_log_analytics_workspace" "log_analytics_workspace" {
        id                              = "/subscriptions/0bf7eeee-63f3-4129-a0bc-b0424afc2633/resourceGroups/rg-kxs-sec-01/providers/Microsoft.OperationalInsights/workspaces/log-kxs-sec-apk-01"
        name                            = "log-kxs-sec-apk-01"
      ~ retention_in_days               = 90 -> 30
        tags                            = {
            "cat_business_unit" = "cloud_engineering"
            "cat_environment"   = "shared"
            "cat_owner"         = "ncuccia"
            "created_by"        = "terraform"
        }
        # (11 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

Plan: 0 to add, 7 to change, 0 to destroy.

```
</details>

<details><summary>Show Diff</summary>

```diff
Index: module.backend.module.storage_account["stcsp2appeastustfstate"].azurerm_storage_account_network_rules.bootstrap
===================================================================
--- module.backend.module.storage_account["stcsp2appeastustfstate"].azurerm_storage_account_network_rules.bootstrap
+++ module.backend.module.storage_account["stcsp2appeastustfstate"].azurerm_storage_account_network_rules.bootstrap
@@ -4,9 +4,8 @@
 id: /subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2appeastustfstate
 ip_rules:
   - 184.162.46.109
   - 184.95.211.82
-  - 20.121.20.179
   - 20.151.110.161
   - 45.61.8.148
   - 75.98.136.253
 private_link_access: []

```

```diff
Index: module.backend.module.storage_account["stcsp2cldeastustfstate"].azurerm_storage_account_network_rules.bootstrap
===================================================================
--- module.backend.module.storage_account["stcsp2cldeastustfstate"].azurerm_storage_account_network_rules.bootstrap
+++ module.backend.module.storage_account["stcsp2cldeastustfstate"].azurerm_storage_account_network_rules.bootstrap
@@ -4,9 +4,8 @@
 id: /subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2cldeastustfstate
 ip_rules:
   - 184.162.46.109
   - 184.95.211.82
-  - 20.121.20.179
   - 20.151.110.161
   - 45.61.8.148
   - 75.98.136.253
 private_link_access: []

```

```diff
Index: module.backend.module.storage_account["stcsp2fweastustfstate"].azurerm_storage_account_network_rules.bootstrap
===================================================================
--- module.backend.module.storage_account["stcsp2fweastustfstate"].azurerm_storage_account_network_rules.bootstrap
+++ module.backend.module.storage_account["stcsp2fweastustfstate"].azurerm_storage_account_network_rules.bootstrap
@@ -4,9 +4,8 @@
 id: /subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2fweastustfstate
 ip_rules:
   - 184.162.46.109
   - 184.95.211.82
-  - 20.121.20.179
   - 20.151.110.161
   - 45.61.8.148
   - 75.98.136.253
 private_link_access: []

```

```diff
Index: module.backend.module.storage_account["stcsp2infraeastustfstate"].azurerm_storage_account_network_rules.bootstrap
===================================================================
--- module.backend.module.storage_account["stcsp2infraeastustfstate"].azurerm_storage_account_network_rules.bootstrap
+++ module.backend.module.storage_account["stcsp2infraeastustfstate"].azurerm_storage_account_network_rules.bootstrap
@@ -4,9 +4,8 @@
 id: /subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2infraeastustfstate
 ip_rules:
   - 184.162.46.109
   - 184.95.211.82
-  - 20.121.20.179
   - 20.151.110.161
   - 45.61.8.148
   - 75.98.136.253
 private_link_access: []

```

```diff
Index: module.backend.module.storage_account["stcsp2neteastustfstate"].azurerm_storage_account_network_rules.bootstrap
===================================================================
--- module.backend.module.storage_account["stcsp2neteastustfstate"].azurerm_storage_account_network_rules.bootstrap
+++ module.backend.module.storage_account["stcsp2neteastustfstate"].azurerm_storage_account_network_rules.bootstrap
@@ -4,9 +4,8 @@
 id: /subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2neteastustfstate
 ip_rules:
   - 184.162.46.109
   - 184.95.211.82
-  - 20.121.20.179
   - 20.151.110.161
   - 45.61.8.148
   - 75.98.136.253
 private_link_access: []

```

```diff
Index: module.monitoring.azurerm_resource_group.main_resource_group
===================================================================
--- module.monitoring.azurerm_resource_group.main_resource_group
+++ module.monitoring.azurerm_resource_group.main_resource_group
@@ -1,8 +1,12 @@
 id: /subscriptions/0bf7eeee-63f3-4129-a0bc-b0424afc2633/resourceGroups/rg-kxs-sec-01
 location: eastus
 name: rg-kxs-sec-01
-tags: {}
+tags:
+  cat_business_unit: cloud_engineering
+  cat_environment: shared
+  cat_owner: ncuccia
+  created_by: terraform
 timeouts:
   create: null
   delete: null
   read: null

```

```diff
Index: module.monitoring.module.loganalytics_workspace.azurerm_log_analytics_workspace.log_analytics_workspace
===================================================================
--- module.monitoring.module.loganalytics_workspace.azurerm_log_analytics_workspace.log_analytics_workspace
+++ module.monitoring.module.loganalytics_workspace.azurerm_log_analytics_workspace.log_analytics_workspace
@@ -8,9 +8,9 @@
 name: log-kxs-sec-apk-01
 primary_shared_key: Q0ITtlgc7gP7hzIgeWdmhadKpXrPRnCP2oDyGoZ5V657NAuTS48x1rsCb0pZ2IRvbqVSimaITRsI6dnORX4Hog==
 reservation_capacity_in_gb_per_day: null
 resource_group_name: rg-kxs-sec-01
-retention_in_days: 90
+retention_in_days: 30
 secondary_shared_key: RXqAfDRjRO8b9tDUnBO7uJWj2ZwO334+ZP25N6PKRhp12sbxW6W432lZB1qabI+9L1gJ1CC4Yaj3/fDnKF7L4w==
 sku: PerGB2018
 tags:
   cat_business_unit: cloud_engineering

```</details>

#### **20-bootstrap** ==> useast
##### Plan: `0` to add, `7` to change, `0` to destroy

<details><summary>Show Plan</summary>

```terraform

Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # module.monitoring.azurerm_resource_group.main_resource_group will be updated in-place
  ~ resource "azurerm_resource_group" "main_resource_group" {
        id       = "/subscriptions/0bf7eeee-63f3-4129-a0bc-b0424afc2633/resourceGroups/rg-kxs-sec-01"
        name     = "rg-kxs-sec-01"
      ~ tags     = {
          + "cat_business_unit" = "cloud_engineering"
          + "cat_environment"   = "shared"
          + "cat_owner"         = "ncuccia"
          + "created_by"        = "terraform"
        }
        # (1 unchanged attribute hidden)

        # (1 unchanged block hidden)
    }

  # module.backend.module.storage_account["stcsp2appeastustfstate"].azurerm_storage_account_network_rules.bootstrap will be updated in-place
  ~ resource "azurerm_storage_account_network_rules" "bootstrap" {
        id                         = "/subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2appeastustfstate"
      ~ ip_rules                   = [
          - "20.121.20.179",
            # (5 unchanged elements hidden)
        ]
        # (4 unchanged attributes hidden)
    }

  # module.backend.module.storage_account["stcsp2cldeastustfstate"].azurerm_storage_account_network_rules.bootstrap will be updated in-place
  ~ resource "azurerm_storage_account_network_rules" "bootstrap" {
        id                         = "/subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2cldeastustfstate"
      ~ ip_rules                   = [
          - "20.121.20.179",
            # (5 unchanged elements hidden)
        ]
        # (4 unchanged attributes hidden)
    }

  # module.backend.module.storage_account["stcsp2fweastustfstate"].azurerm_storage_account_network_rules.bootstrap will be updated in-place
  ~ resource "azurerm_storage_account_network_rules" "bootstrap" {
        id                         = "/subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2fweastustfstate"
      ~ ip_rules                   = [
          - "20.121.20.179",
            # (5 unchanged elements hidden)
        ]
        # (4 unchanged attributes hidden)
    }

  # module.backend.module.storage_account["stcsp2infraeastustfstate"].azurerm_storage_account_network_rules.bootstrap will be updated in-place
  ~ resource "azurerm_storage_account_network_rules" "bootstrap" {
        id                         = "/subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2infraeastustfstate"
      ~ ip_rules                   = [
          - "20.121.20.179",
            # (5 unchanged elements hidden)
        ]
        # (4 unchanged attributes hidden)
    }

  # module.backend.module.storage_account["stcsp2neteastustfstate"].azurerm_storage_account_network_rules.bootstrap will be updated in-place
  ~ resource "azurerm_storage_account_network_rules" "bootstrap" {
        id                         = "/subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2neteastustfstate"
      ~ ip_rules                   = [
          - "20.121.20.179",
            # (5 unchanged elements hidden)
        ]
        # (4 unchanged attributes hidden)
    }

  # module.monitoring.module.loganalytics_workspace.azurerm_log_analytics_workspace.log_analytics_workspace will be updated in-place
  ~ resource "azurerm_log_analytics_workspace" "log_analytics_workspace" {
        id                              = "/subscriptions/0bf7eeee-63f3-4129-a0bc-b0424afc2633/resourceGroups/rg-kxs-sec-01/providers/Microsoft.OperationalInsights/workspaces/log-kxs-sec-apk-01"
        name                            = "log-kxs-sec-apk-01"
      ~ retention_in_days               = 90 -> 30
        tags                            = {
            "cat_business_unit" = "cloud_engineering"
            "cat_environment"   = "shared"
            "cat_owner"         = "ncuccia"
            "created_by"        = "terraform"
        }
        # (11 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

Plan: 0 to add, 7 to change, 0 to destroy.

```
</details>

<details><summary>Show Diff</summary>

```diff
Index: module.backend.module.storage_account["stcsp2appeastustfstate"].azurerm_storage_account_network_rules.bootstrap
===================================================================
--- module.backend.module.storage_account["stcsp2appeastustfstate"].azurerm_storage_account_network_rules.bootstrap
+++ module.backend.module.storage_account["stcsp2appeastustfstate"].azurerm_storage_account_network_rules.bootstrap
@@ -4,9 +4,8 @@
 id: /subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2appeastustfstate
 ip_rules:
   - 184.162.46.109
   - 184.95.211.82
-  - 20.121.20.179
   - 20.151.110.161
   - 45.61.8.148
   - 75.98.136.253
 private_link_access: []

```

```diff
Index: module.backend.module.storage_account["stcsp2cldeastustfstate"].azurerm_storage_account_network_rules.bootstrap
===================================================================
--- module.backend.module.storage_account["stcsp2cldeastustfstate"].azurerm_storage_account_network_rules.bootstrap
+++ module.backend.module.storage_account["stcsp2cldeastustfstate"].azurerm_storage_account_network_rules.bootstrap
@@ -4,9 +4,8 @@
 id: /subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2cldeastustfstate
 ip_rules:
   - 184.162.46.109
   - 184.95.211.82
-  - 20.121.20.179
   - 20.151.110.161
   - 45.61.8.148
   - 75.98.136.253
 private_link_access: []

```

```diff
Index: module.backend.module.storage_account["stcsp2fweastustfstate"].azurerm_storage_account_network_rules.bootstrap
===================================================================
--- module.backend.module.storage_account["stcsp2fweastustfstate"].azurerm_storage_account_network_rules.bootstrap
+++ module.backend.module.storage_account["stcsp2fweastustfstate"].azurerm_storage_account_network_rules.bootstrap
@@ -4,9 +4,8 @@
 id: /subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2fweastustfstate
 ip_rules:
   - 184.162.46.109
   - 184.95.211.82
-  - 20.121.20.179
   - 20.151.110.161
   - 45.61.8.148
   - 75.98.136.253
 private_link_access: []

```

```diff
Index: module.backend.module.storage_account["stcsp2infraeastustfstate"].azurerm_storage_account_network_rules.bootstrap
===================================================================
--- module.backend.module.storage_account["stcsp2infraeastustfstate"].azurerm_storage_account_network_rules.bootstrap
+++ module.backend.module.storage_account["stcsp2infraeastustfstate"].azurerm_storage_account_network_rules.bootstrap
@@ -4,9 +4,8 @@
 id: /subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2infraeastustfstate
 ip_rules:
   - 184.162.46.109
   - 184.95.211.82
-  - 20.121.20.179
   - 20.151.110.161
   - 45.61.8.148
   - 75.98.136.253
 private_link_access: []

```

```diff
Index: module.backend.module.storage_account["stcsp2neteastustfstate"].azurerm_storage_account_network_rules.bootstrap
===================================================================
--- module.backend.module.storage_account["stcsp2neteastustfstate"].azurerm_storage_account_network_rules.bootstrap
+++ module.backend.module.storage_account["stcsp2neteastustfstate"].azurerm_storage_account_network_rules.bootstrap
@@ -4,9 +4,8 @@
 id: /subscriptions/34dbb730-7f89-4e5b-9102-dbc8fbb3b966/resourceGroups/rg-kxs-bootstrap-eastus-01/providers/Microsoft.Storage/storageAccounts/stcsp2neteastustfstate
 ip_rules:
   - 184.162.46.109
   - 184.95.211.82
-  - 20.121.20.179
   - 20.151.110.161
   - 45.61.8.148
   - 75.98.136.253
 private_link_access: []

```

```diff
Index: module.monitoring.azurerm_resource_group.main_resource_group
===================================================================
--- module.monitoring.azurerm_resource_group.main_resource_group
+++ module.monitoring.azurerm_resource_group.main_resource_group
@@ -1,8 +1,12 @@
 id: /subscriptions/0bf7eeee-63f3-4129-a0bc-b0424afc2633/resourceGroups/rg-kxs-sec-01
 location: eastus
 name: rg-kxs-sec-01
-tags: {}
+tags:
+  cat_business_unit: cloud_engineering
+  cat_environment: shared
+  cat_owner: ncuccia
+  created_by: terraform
 timeouts:
   create: null
   delete: null
   read: null

```

```diff
Index: module.monitoring.module.loganalytics_workspace.azurerm_log_analytics_workspace.log_analytics_workspace
===================================================================
--- module.monitoring.module.loganalytics_workspace.azurerm_log_analytics_workspace.log_analytics_workspace
+++ module.monitoring.module.loganalytics_workspace.azurerm_log_analytics_workspace.log_analytics_workspace
@@ -8,9 +8,9 @@
 name: log-kxs-sec-apk-01
 primary_shared_key: Q0ITtlgc7gP7hzIgeWdmhadKpXrPRnCP2oDyGoZ5V657NAuTS48x1rsCb0pZ2IRvbqVSimaITRsI6dnORX4Hog==
 reservation_capacity_in_gb_per_day: null
 resource_group_name: rg-kxs-sec-01
-retention_in_days: 90
+retention_in_days: 30
 secondary_shared_key: RXqAfDRjRO8b9tDUnBO7uJWj2ZwO334+ZP25N6PKRhp12sbxW6W432lZB1qabI+9L1gJ1CC4Yaj3/fDnKF7L4w==
 sku: PerGB2018
 tags:
   cat_business_unit: cloud_engineering

```</details>



---
<!-- TFCHANGES END -->

> This comment was generated by [Terraform PR Reporter](https://github.com/Kinaxis/terraform-pr-reporter) - Action run [`#NaN`](https://github.com/Codertocat/Hello-World/actions/runs/NaN) - Commit undefined

**Review by OppilaalDaniel — CHANGES_REQUESTED** [review]
Reload failed.
Log:
ReloadID: 651e4f6332465c2bedda213f
Started loading data
(A detailed script progress log can be downloaded when the reload is finished)
Data has not been loaded. Please correct the error and try loading again.
The following error occurred:
Unknown statement: Nobody reads
The error occurred here:
Nobody reads

**Review by OppilaalDaniel — CHANGES_REQUESTED** [review]
Reload failed.
Log:
ReloadID: 651e50b19db56064f835bc0a
Started loading data
(A detailed script progress log can be downloaded when the reload is finished)
Data has not been loaded. Please correct the error and try loading again.
The following error occurred:
Unknown statement: Nobody reads
The error occurred here:
Nobody reads

**Review by ftf-spn — CHANGES_REQUESTED** [review]
ReloadID: 65560deac7d4e9dd5f54135f
Started loading data
(A detailed script progress log can be downloaded when the reload is finished)
Connected
Fact_Returns 6,214 Lines fetched

App saved
Finished successfully

**Review by ftf-spn — APPROVED** [review]

**Review by Stan-Decoene-element61 — CHANGES_REQUESTED** [review]
Load script test failed.
Log:
ReloadID: 65f819d8eaa235f9543d4be9
Started loading data
(A detailed script progress log can be downloaded when the reload is finished)
Data has not been loaded. Please correct the error and try loading again.
The following error occurred:
Unknown statement: Nobody reads
The error occurred here:
Nobody reads

**Review by Onkar2u — CHANGES_REQUESTED** [review]
Load script test failed.
Log:
ReloadID: 66ea6fae1b3109b1c84c0911
Started loading data
(A detailed script progress log can be downloaded when the reload is finished)
Data has not been loaded. Please correct the error and try loading again.
The following error occurred:
Unknown statement: Nobody reads
The engine error code: EDC_ERROR:11002
The error occurred here:
Nobody reads

**Review by ChrFreese — CHANGES_REQUESTED** [review]
Reload failed.
Log:
ReloadID: 6937301ae114ea1450e8ea76
Started loading data
(A detailed script progress log can be downloaded when the reload is finished)
Data has not been loaded. Please correct the error and try loading again.
The following error occurred:
Unknown statement: Nobody reads
The engine error code: Syntax error:11002
The error occurred here:
Nobody reads

**Review by SudiptaDuttaIFZ — CHANGES_REQUESTED** [review]
Reload failed.
Log:
ReloadID: 698c48f157c7ba58d0577a49
Started loading data
(A detailed script progress log can be downloaded when the reload is finished)
Data has not been loaded. Please correct the error and try loading again.
The following error(s) occurred:
Unknown statement: Nobody reads

**Review by lindibeetge — CHANGES_REQUESTED** [review]
Load script test failed.
Log:
ReloadID: 6a04e8a30f51bf589c8b440c
Started loading data
(A detailed script progress log can be downloaded when the reload is finished)
Data has not been loaded. Please correct the error and try loading again.
The following error(s) occurred:
Unknown statement: Nobody reads

**Review by lindibeetge — CHANGES_REQUESTED** [review]
Load script test failed.
Log:
ReloadID: 6a04e9620f51bf589c8b44b7
Started loading data
(A detailed script progress log can be downloaded when the reload is finished)
Data has not been loaded. Please correct the error and try loading again.
The following error(s) occurred:
Unknown statement: Nobody reads

**Pull request closed without merging**